examples/pca9635/pca9635_main.c

```diff
@@ -59,7 +59,7 @@ int main(int argc, FAR char *argv[])
   int fd;
   int ret;
 
-  fd = open(CONFIG_EXAMPLES_PCA9635_DEVNAME, O_RDONLY);
+  fd = open(CONFIG_EXAMPLES_PCA9635_DEVNAME, 0);
```

Comment:
Should we change O_RDONLY to O_WRONLY? PWMIOC_SETLED_BRIGHTNESS need the write permission.

Reply:
Done